certification/internal/client/client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	operatorv1 "github.com/operator-framework/api/pkg/operators/v1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	scheme                   = runtime.NewScheme()
+	operatorV1SchemeGV       = schema.GroupVersion{Group: "operators.coreos.com", Version: "v1"}
+	OperatorV1Alpha1SchemeGV = schema.GroupVersion{Group: "operators.coreos.com", Version: "v1alpha1"}
+)
+
+func addKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
Should this function have a return type since we are only ever returning nil? or is the intent to add more error handling later?

Reply:
Thank you Adam for taking the time to review my code.

This function is being passed as an argument to XXXXClient in each client file (catalogsource.go, operatorgroup.go, etc). I checked the functions that are being called and they do not have return value (https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#AddToGroupVersion and https://github.com/kubernetes/apimachinery/blob/a644435e2c133d990b624858d9c01985d7f59adf/pkg/runtime/scheme.go#L162).

That being said, none of these function calls return anything and that's why I defined this method to return nothing as well.